TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -48,25 +48,65 @@ function Execute-SqlQueryDeployment
         # Import SQLPS Module
         Import-SqlPs
 
-        $scriptArgument = "Invoke-Sqlcmd -ServerInstance `"$serverName`" -Database `"$databaseName`" -InputFile `"$sqlFile`" "
-        $scriptArgument += $additionalArguments
+        $spaltArguments = @{
+            ServerInstance=$serverName
+            Database=$databaseName
+            InputFile=$sqlFile
+        }
+
+        # Process Additional arguments 
+        if($additionalArguments) 
+        {
+            $args = $additionalArguments.Split()
+            $argScope = $null
+            foreach($arg in $args) {
+                $arg = $arg.Trim()
+                if($arg)
+                {
```

Comment:
it is done with "Get-ParsedSessionVariables" which is commandlet not part of dtl-sdk